docs/user-guides/finetuner.md

```diff
@@ -0,0 +1,144 @@
+# Fine-tune Models
+
+Although CLIP-as-service has provided you a list of pre-trained models, you can also train your models. 
+This guide will show you how to use [Finetuner](https://finetuner.jina.ai) to fine-tune models and use them in CLIP-as-service.
+
+For installation and basic usage of Finetuner, please refer to [Finetuner website](https://finetuner.jina.ai).
+
+## Prepare Training Data
+
+Finetuner accepts training data and evaluation data in the form of [`DocumentArray`](https://docarray.jina.ai/fundamentals/documentarray/).
+The training data for CLIP is a list of (text, image) pairs.
+Each pair is store in a [`Document`](https://docarray.jina.ai/fundamentals/document/) which wraps two [`chunks`](https://docarray.jina.ai/fundamentals/document/nested/) with the `image` and `text` modality.
+You can push the resulting [`DocumentArray`](https://docarray.jina.ai/fundamentals/documentarray/) to the cloud using the [`.push`](https://docarray.jina.ai/api/docarray.array.document/?highlight=push#docarray.array.document.DocumentArray.push) method.
+A sample to construct and push the training data is shown below.
+
+```python
+from docarray import Document, DocumentArray
+
+train_da = DocumentArray(
+    [
+        Document(
+            chunks=[
+                Document(
+                    content='pencil skirt slim fit available for sell',
+                    modality='text',
+                ),
+                Document(
+                    uri='https://...skirt-1.png',
+                    modality='image',
+                ),
+            ],
+        ),
+    ]
+)
+train_da.push('name-of-train-data')
+```
+
+## Start Finetuner
+
+You may now create and run a fine-tuning job after login to Jina ecosystem.
+For simplicity, we directly use `clip-fashion-train-data` and `clip-fashion-eval-data` provided by Finetuner.
```

Comment:
The pre-defined dataset `clip-fashion` is an example dataset. Not **For simplicity**. This sentence will mislead the reader.